spanner/changestreams/src/main/java/com/example/spanner/changestreams/ChangeStreamSample.java

```diff
@@ -0,0 +1,255 @@
+ * 5. Drops the created table and change stream.
+ */
+public class ChangeStreamSample {
+  private static final long TIMEOUT_MINUTES = 2;
+
+  public static void main(String... args) {
+    if (args.length != 3) {
+      System.err.println("Usage: ChangeStreamSample "
+          + "<instance_id> "
+          + "<database_id> "
+          + "<prefix>");
+      return;
+    }
+
+    final String instanceId = args[0];
+    final String databaseId = args[1];
+    final String prefix = args[2];
+    final String tableName = prefix + "_Singers";
+    final String changeStreamName = prefix + "_ChangeStreamSingers";
+    SpannerOptions options = SpannerOptions.newBuilder().build();
+    Spanner spanner = options.getService();
+    DatabaseAdminClient dbAdminClient = spanner.getDatabaseAdminClient();
+    DatabaseClient dbClient = spanner.getDatabaseClient(
+        DatabaseId.of(options.getProjectId(), instanceId, databaseId));
+
+    try {
+      createChangeStream(dbAdminClient, instanceId, databaseId, tableName, changeStreamName);
+      queryChangeStream(dbClient, tableName, changeStreamName);
+      dropChangeStream(dbAdminClient, instanceId, databaseId, tableName, changeStreamName);
+    } catch (Exception e) {
+      System.out.println(e);
+    } finally {
+      spanner.close();
+    }
+  }
+
+  public static void updateDdl(
+      DatabaseAdminClient dbAdminClient, String instanceId, String databaseId,
+      List<String> statements) throws Exception {
+    OperationFuture<Void, UpdateDatabaseDdlMetadata> op = dbAdminClient
+        .updateDatabaseDdl(
+          instanceId,
+          databaseId,
+          statements,
+            null
+        );
+
+    op.get(TIMEOUT_MINUTES, TimeUnit.MINUTES);
+  }
+
+  // [START change_stream_sample_create_change_stream]
+  static void createChangeStream(
+      DatabaseAdminClient dbAdminClient, String instanceId, String databaseId, String tableName,
+      String changeStreamName) throws Exception {
+    System.out.println(
+        String.format("Updating database DDL to create table %s and change stream %s.",
+          tableName, changeStreamName));
+
+    updateDdl(
+        dbAdminClient,
+        instanceId,
+        databaseId,
+        Arrays.asList(
+          String.format("CREATE TABLE %s ("
+            + "  SingerId   INT64 NOT NULL,"
+            + "  FirstName  STRING(1024),"
+            + "  LastName   STRING(1024)"
+            + ") PRIMARY KEY (SingerId)", tableName),
+          String.format("CREATE CHANGE STREAM %s FOR %s", changeStreamName, tableName)
+      ));
+  }
+  // [END change_stream_sample_create_change_stream]
+
+  // [START change_stream_sample_query_change_stream]
+  public static void queryChangeStream(DatabaseClient dbClient, String tableName,
+                                       String changeStreamName) {
+    // Insert test data into the table.
+    System.out.println(
+        String.format("Inserting rows "
+          + "(1, Melissa, Garcia) and "
```

Comment:
Should we use real names here or just something like "User 1 First Name", "User 1 Last Name"?

Reply:
Changed.

Reply:
It seems we forgot to change this String?